src/main/java/cz/inventi/kpj/spring/springintroduction/persistence/GreetingConstantRepositoryImpl.java

```diff
@@ -0,0 +1,13 @@
+package cz.inventi.kpj.spring.springintroduction.persistence;
+
+import org.springframework.stereotype.Component;
+
+import lombok.Data;
+
+@Data
```

Comment:
Data annotation doesn't make sense here. You don't have any attributes here.
In general, I don't use @Data annotation on the service. I don't want to make dependencies visible outside the class. (You don't want to allow call GreetingConstantRepositoryImpl.getSomeDependency();)